src/flex-example-app-android.xml

```diff
@@ -72,6 +76,14 @@
                     <activity android:name="com.vungle.publisher.FullScreenAdActivity" android:configChanges="keyboardHidden|orientation|screenSize" android:theme="@android:style/Theme.NoTitleBar.Fullscreen"    />
                     <service android:name="com.vungle.publisher.VungleService" android:exported="false"/>
 
+
+<activity android:name="com.inmobi.rendering.InMobiAdActivity" android:configChanges="keyboardHidden|orientation|keyboard|smallestScreenSize|screenSize" android:theme="@android:style/Theme.Translucent.NoTitleBar" android:hardwareAccelerated="true" /><receiver android:name="com.inmobi.commons.core.utilities.uid.ImIdShareBroadCastReceiver" android:enabled="true" android:exported="true" >
+  <intent-filter>
+    <action android:name="com.inmobi.share.id" />
+  </intent-filter>
+</receiver>
+                    <activity android:name="com.inmobi.androidsdk.IMBrowserActivity" android:configChanges="keyboardHidden|orientation|keyboard|smallestScreenSize|screenSize" android:theme="@android:style/Theme.Translucent.NoTitleBar" android:hardwareAccelerated="true" />
```

Comment:
As I can remember I had some problems with displaying an activity or found it in their logs while testing app on android. 
btw atm Test suite shows me "network not enabled". don't know why...